tensorboard/backend/application.py

```diff
@@ -849,3 +876,34 @@ def make_plugin_loader(plugin_spec):
         if issubclass(plugin_spec, base_plugin.TBPlugin):
             return base_plugin.BasicLoader(plugin_spec)
     raise TypeError("Not a TBLoader or TBPlugin subclass: %r" % (plugin_spec,))
+
+
+class ExperimentalPlugin(object):
```

Comment:
Probably worth mentioning on the docstring that this marker class is itself experimental (i.e. that the class itself and how it's interpreted may have breaking changes in the future, so there are no guarantees around using it).

Reply:
Done.